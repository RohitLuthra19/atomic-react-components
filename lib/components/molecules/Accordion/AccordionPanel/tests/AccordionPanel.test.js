import React from 'react';
import { shallow } from 'enzyme';

import { AccordionPanelVanilla } from '../index';

describe('<AccordionPanel />', () => {
  let AccordionPanelComponent = '';
  beforeEach(() => {
    AccordionPanelComponent = shallow(<AccordionPanelVanilla>Test</AccordionPanelVanilla>);
  });

  test('should render correctly', () => {
    expect(AccordionPanelComponent).toMatchSnapshot();
  });
});
