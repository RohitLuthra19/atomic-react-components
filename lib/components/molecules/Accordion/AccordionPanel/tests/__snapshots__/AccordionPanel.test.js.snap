// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccordionPanel /> should render correctly 1`] = `
<div
  className=""
>
  Test
</div>
`;
